src/backoff.test.ts

```diff
@@ -0,0 +1,23 @@
+import { createGetBackoff } from "backoff";
+
+describe('getBackoff', () => {
+  [
+    { n: 0, p: 1 },
+    { n: 9, p: 1 },
+    { n: 10, p: 2 },
+    { n: 14, p: 2 },
+    { n: 15, p: 3 },
+    { n: 16, p: 3 },
+    { n: 17, p: 4 },
+    { n: 50, p: 4 },
+  ].forEach((element) => {
```

Comment:
Yup!  Except `it.each` isn't normally available to us backenders.  And I think this is clearer anyway, using string interp instead of having to learn and remember the `%` token system.